plugins/modules/aci_l4l7_policy_based_redirect.py

```diff
@@ -0,0 +1,387 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Tim Cragg (@timcragg)
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l4l7_policy_based_redirect
+short_description: Manage L4-L7 Policy Based Redirection (vns:SvcRedirectPol)
```

Comment:
should we add policies?
```suggestion
short_description: Manage L4-L7 Policy Based Redirection Policies (vns:SvcRedirectPol)
```